.github/workflows/ci.yaml

```diff
@@ -60,28 +60,44 @@ jobs:
         pip install -r tests/requirements-test.txt
         python -m pytest -v -p no:warnings
         deactivate
+  deployment-tests:
+    runs-on: self-hosted
+    strategy:
+      matrix:
```

Comment:
reading up on actions matrix, very cool! could be helpful way of thinking about future environments and profiles

Reply:
+1